### DISCUSSION TITLE
is that possible to make requests delya in certain situations?

### DISCUSSION THREAD

**Discussion by Reg93** [reduxjs/redux-toolkit#4647] [Q&A] [answered]
I would like to make something like delay in basic api in situations when access toke is expired and it requires to send refresh. Now if access is expired every request makes its own token refresh and instead of 3 i get 6 requests.

So i would like first requests to send refresh token and next requests to await a bit and to use new access token

**Comment by EskiMojo14** [comment, marked as answer]
see [preventing multiple unauthorized errors](https://redux-toolkit.js.org/rtk-query/usage/customizing-queries#preventing-multiple-unauthorized-errors) from our docs.

**Reply by Reg93** [reply]
Thank you!!!
